ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master-presubmits.yaml

```diff
@@ -402,6 +402,74 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-operator,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: build02
+    context: ci/prow/e2e-gcp-serial
```

Comment:
Elsewhere I see `context: ci/prow/k8s-e2e-gcp-serial` (and `mountPath: /usr/local/k8s-e2e-gcp-serial-cluster-profile`, `--target=k8s-e2e-gcp-serial`, and so on); should this have `k8s` in the name?

Reply:
Hm, what other repos where you looking at that had `k8s-e2e-gcp-serial`? Kube-component repos?

None of the definitions in our repos use the `ci/prow/k8s-*` pattern, so I think this is fine.

Reply:
When I ran `git -C ~/src/github.com/openshift/release grep -e 'context: ci/prow/e2e-gcp-serial'`, the only results I saw were for cluster-ingress-operator and coredns (I had your PR checked out locally).  When I ran `git -C ~/src/github.com/openshift/release grep -e 'context: ci/prow/k8s-e2e-gcp-serial'`, I saw results for cluster-kube-apiserver-operator and kubernetes.  Maybe it's a kube-component thing and not an e2e-serial thing.

Reply:
When I ran `git -C ~/src/github.com/openshift/release grep -e 'context: ci/prow/e2e-gcp-serial'`, the only results I saw were for cluster-ingress-operator and coredns (I had your PR checked out locally).  When I ran `git -C ~/src/github.com/openshift/release grep -e 'context: ci/prow/k8s-e2e-gcp-serial'`, I saw results for cluster-kube-apiserver-operator and kubernetes.  Maybe it's a kube-component thing and not an e2e-serial thing.

Reply:
No it should not have k8s in the name.  That's only for openshift/kubernetes